src/Google.Api.Gax/GaxPreconditions.cs

```diff
@@ -237,13 +338,7 @@ public static void CheckArgument<T>(bool condition, string paramName, string for
         /// </summary>
         /// <param name="value">The value to check.</param>
         /// <param name="paramName">The name of the parameter whose value is being tested.</param>
-        internal static TimeSpan CheckNonNegativeDelay(TimeSpan value, string paramName)
-        {
-            if (value < TimeSpan.Zero)
-            {
-                throw new ArgumentOutOfRangeException(nameof(value), value, "Delay must not be negative");
-            }
-            return value;
-        }
+        internal static TimeSpan CheckNonNegativeDelay(TimeSpan value, string paramName) =>
```

Comment:
Possibly rename to "TimeSpan" or "Duration" rather than "Delay"? This might not be a delay. (Ditto the comment)